.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,7 +14,7 @@ Please remove this line and everything above and fill the following sections:
 ### Change description ###
 <!-- Include what your change is here, any testing you have done, etc -->
 
-**Does this PR introduce a breaking change?** <!--(check one with "x") -->
+<br>**Does this PR introduce a breaking change?** <!--(check one with "x") -->
```

Comment:
you shouldn't really use br in markdown, just use whitespace correctly

Reply:
Well then you end up having loads of empty lines and never quite know how many you need. I personally prefer the <br> tag since its way neater.